src/lib/database/database.config.ts

```diff
@@ -20,8 +20,7 @@ export const config: ConnectionOptions = {
 		migrationsDir: 'src/lib/database/migrations',
 		subscribersDir: 'src/lib/database/subscribers'
 	},
-	namingStrategy: new SnakeNamingStrategy(),
-	logging: DEV
```

Comment:
Really not entirely happy with this but w/e I guess. It's never bothered me but maybe that's because I basically never use console.log to debug, instead use breakpoints. And so should you 👀

Reply:
I'm not using console.log to debug, but not being able to see command usage logs, as well as `console.time`'s, is annoying.